### DISCUSSION TITLE
https://greendolphindance.com/2023/10/09/%E5%A1%94%E7%BD%97%E7%89%8C%E3%80%81%E7%94%9F%E5%91%BD%E7%81%B5%E6%95%B0%E3%80%81%E6%98%9F%E5%BA%A7%E5%92%8CMBTI/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#7] [General]
# https://greendolphindance.com/2023/10/09/%E5%A1%94%E7%BD%97%E7%89%8C%E3%80%81%E7%94%9F%E5%91%BD%E7%81%B5%E6%95%B0%E3%80%81%E6%98%9F%E5%BA%A7%E5%92%8CMBTI/

Back to the rough ground!

https://greendolphindance.com/2023/10/09/%E5%A1%94%E7%BD%97%E7%89%8C%E3%80%81%E7%94%9F%E5%91%BD%E7%81%B5%E6%95%B0%E3%80%81%E6%98%9F%E5%BA%A7%E5%92%8CMBTI/

**Comment by redsherry** [comment]
您好高产！星盘我觉得有的还挺准的，而且解读起来也非常复杂。不光要看什么星是什么座，还要看什么宫落入什么座。而且根据星体变化，星座运势也会改变。我现在每周都会看占星运势……

**Reply by greendolphindance** [reply]
啊，那可能是我出生时间没记对？反正我的看起来很不准啊。星盘有可以避免同时间生日的人有着相同性格/运势的机制吗？

**Reply by redsherry** [reply]
出生在同一个小时的人的星盘应该都是一样的吧）但是可以做不同的解释。毕竟也有上升星座啊，每个人太阳和上升的比例可能不太一样

**Reply by greendolphindance** [reply]
但是上升星座也是由出生时间和出生地决定的吧？

**Reply by redsherry** [reply]
是的。只是有解释的空间。
